### PR TITLE
fix(cookie-policy) Typo in check function

### PR DESCRIPTION
**Pull request by catfishbluefish**
Fixes a typo in the function checking if the cookie policy cookie is set

### CODE REVIEW

File: app/_assets/javascripts/app.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ $(function () {
   // Cookie Policy
   // Check for cookie
   if (!document.cookie.split(';').filter(function (item) {
-    return item.indexOf('cookie-policy=') >= 0
+    return item.indexOf('cookie-policy') >= 0
```
[the remark this answers is not in the document]
Comment (catfishbluefish) [reply app/_assets/javascripts/app.js]: as far as I can tell, not easily. The search through the array of document.cookies seems to equal a different string than what is directly assigned in line 57. If I set 
`Cookie = 'cookie-policy; expires=' + CookieDate.toGMTString() + ';path=/'` 
`  return item.indexOf(Cookie) >= 0` returns false

[the remark this answers is not in the document]
Comment (darrenjennings) [reply app/_assets/javascripts/app.js]: I mean storing the name of the cookie in a variable to be used in these functions. It looks like we are not storing the name of the cookie on the document, but rather the cookie is now a blank name with the value="cookie-policy". In the [original commit](https://github.com/Kong/getkong.org/commit/c4c842a06a6e87191e4f2062d07bef4455bd6fad#diff-dd1ca575b0f23e822cc1e59e8d7b03e9), we had the name of the cookie set to 'cookie-policy', but with a blank value. Any reason for the change? I recommend that the cookie have a name so it's easy to tell if it's false/true or just not even set.

**Comment by catfishbluefish** [comment]
This is my first time working with cookies - this new format seems to be a better way of handling it

File: app/_assets/javascripts/app.js [hunk 2]
```diff
@@ -36,26 +36,42 @@ $(function () {
...
+      if (c.indexOf(name) === 0) {
+        return c.substring(name.length, c.length)
+      }
+    }
+    return ''
+  }
+
+  function setCookie (cname, cvalue) {
+    var CookieDate = new Date()
+    CookieDate.setFullYear(CookieDate.getFullYear() + 1)
```
Comment (darrenjennings) [inline app/_assets/javascripts/app.js]: Looking good. Any reason we're limiting this cookie to one year? I see that the konghq.com cookie is set to never expire.

Comment (catfishbluefish) [reply app/_assets/javascripts/app.js]: I've gone back and forth with people on how long it should be set. I will most likely update Konghq to also be a year.

**Pull request merged**